src/xb-builder-source-ctx-private.h

```diff
@@ -12,7 +12,8 @@
 
 G_BEGIN_DECLS
 
-XbBuilderSourceCtx *xb_builder_source_ctx_new		(GInputStream		*istream);
+XbBuilderSourceCtx *xb_builder_source_ctx_new		(GFile			*file,
+							 GInputStream		*istream);
```

Comment:
Maybe we add xb_builder_source_ctx_file_new() instead?

Reply:
Are you sure? I’d agree if `file` and `istream` were mutually exclusive, but `istream` is always required here, and `file` is optional (nullable). Making them separate functions would complicate the call in `xb-builder-source.c` — it’d go from
```
g_autoptr(XbBuilderSourceCtx) ctx = xb_builder_source_ctx_new (file, priv->istream);
```
to
```
g_autoptr(XbBuilderSourceCtx) ctx = NULL;
if (file != NULL)
  ctx = xb_builder_source_ctx_file_new (file, priv->istream);
else
  ctx = xb_builder_source_ctx_new (priv->istream);
```

Reply:
> Are you sure

I'm never sure! Maybe you're right.